kubernetes/test-infra/gke/main.tf

```diff
@@ -3,11 +3,11 @@ variable "kubernetes_version" {
 }
 
 variable "workers_count" {
-  default = "1"
```

Comment:
I would tweak this value via `-var` cmd line argument from a GH action script variable. I personally also use the configuration in `kubernetes/test-infra/gke` to spin up my own test envs for local development and I'd be nice to keep the size of the two independently adjustable.